packages/web-components/src/horizontalbarchart/horizontalbarchart.styles.ts

```diff
@@ -75,13 +77,54 @@ export const styles: ElementStyles = css`
     opacity: 0;
     z-index: 999;
   }
+  .bar {
+    opacity: 1;
+  }
   .chartTitle {
     display: flex;
     justify-content: space-between;
     text-overflow: ellipsis;
     overflow: hidden;
     white-space: nowrap;
     display: block;
-    color: theme.palette.neutralPrimary;
+    font-size: 14px;
+    font-weight: 400px;
+    line-height: 16px;
+    color: rgb(50, 49, 48);
+    margin-bottom: 5px;
+  }
+  .legendcontainer {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding-top: 16px;
+    width: 90%;
+    align-items: center;
+    margin: -8px 0 0 -8px;
+  }
+  .legend {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    border: none;
+    padding: 8px;
+    background: none;
+    text-transform: capitalize;
+  }
+  .legendRect {
+    width: 12px;
+    height: 12px;
+    margin-right: 8px;
+    border: 1px solid;
+  }
+  .legendText {
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase200};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase200};
```

Comment:
replace these group with typography defined in this file. https://github.com/microsoft/fluentui/blob/master/packages/web-components/src/styles/partials/typography.partials.ts

For eg all the font related styles here can be replaced with this
https://github.com/microsoft/fluentui/blob/263ead68a05ac23499be137b41e02a5579522c08/packages/web-components/src/styles/partials/typography.partials.ts#L56

Reply:
Okay got it. But this works as a group. In case we need different styles within one group?

Reply:
we can override any specific style element if needed. Though it would be a corner case. We want to follow the standard design tokens everywhere.

Reply:
Okay got it.